timeless/restaurants/table_shapes/forms.py

```diff
@@ -1,10 +1,37 @@
 """Forms for table shapes blueprint in order to support CRUD operations"""
+from flask_wtf.file import FileRequired, FileAllowed, FileField
+
 from timeless import forms
 from timeless.restaurants import models
+from timeless.uploads import IMAGES
 
 
 class TableShapeForm(forms.ModelForm):
     """Base form for table shape"""
 
+    picture = FileField(
+        "Table Shape Image",
+        validators=[FileRequired(), FileAllowed(IMAGES, "Images only!")])
+
     class Meta:
         model = models.TableShape
+        exclude = ("picture",)
+
+    def __init__(self, form, files):
+        super(TableShapeForm, self).__init__(form)
+        self._file = files.get("picture")
+
+    def save(self, commit=True):
+        """
+        Saves uploaded image
+        :param commit: To commit flag
+        :return: Saved instance
+        @todo #205:30min Lets save only filename which is a static
```

Comment:
@ekondrashev aren't you saving the filename already? what is supposed to be done by the one who gets this todo?

Reply:
@ammaratef45 currently it saves url, with address and path hardcoded, in fact we need to call `IMAGES.url` each time we need to render it in UI, since it is a dynamic value